lighthouse/constants.py

```diff
@@ -10,9 +10,36 @@
 FIELD_PLATE_BARCODE = "plate_barcode"
 FIELD_COG_BARCODE = "cog_barcode"
 FIELD_DATE_TESTED = "Date Tested"
+FIELD_CQ_1 = "ch1_cq"
+FIELD_CQ_2 = "ch2_cq"
+FIELD_CQ_3 = "ch3_cq"
 
 # MLWH lighthouse samples table field names
 MLWH_LH_SAMPLE_ROOT_SAMPLE_ID ="root_sample_id"
 MLWH_LH_SAMPLE_COG_UK_ID = "cog_uk_id"
 MLWH_LH_SAMPLE_RNA_ID = "rna_id"
-MLWH_LH_SAMPLE_RESULT = "result"
\ No newline at end of file
+MLWH_LH_SAMPLE_RESULT = "result"
+
+CT_VALUE_LIMIT = 30
```

Comment:
float? examples have 8dp e.g. 30.0
how do datatypes work in python

Reply:
> Python fully supports mixed arithmetic: when a binary arithmetic operator has operands of different numeric types, the operand with the “narrower” type is widened to that of the other, where integer is narrower than floating point, which is narrower than complex. A comparison between numbers of different types behaves as though the exact values of those numbers were being compared.

 (https://docs.python.org/3/library/stdtypes.html)

Seems OK :) I'll change the numbers in the tests to floats